src/main/java/com/walking/intensive/chapter1/task5/Task5.java

```diff
@@ -1,128 +1,134 @@
+        }
+
+        double[] segmentsMultiplication = new double[3];
+        segmentsMultiplication[0] = ((a * c) / (a + b)) * ((b * c) / (a + b));
+        segmentsMultiplication[1] = ((a * b) / (a + c)) * ((b * c) / (a + c));
+        segmentsMultiplication[2] = ((a * c) / (b + c)) * ((a * b) / (b + c));
+
+        double[] bisectorsMassive = new double[3];
+        bisectorsMassive[0] = Math.sqrt(a * b - segmentsMultiplication[0]);
+        bisectorsMassive[1] = Math.sqrt(a * c - segmentsMultiplication[1]);
+        bisectorsMassive[2] = Math.sqrt(b * c - segmentsMultiplication[2]);
+        Arrays.sort(bisectorsMassive);
+
+        return bisectorsMassive;
     }
 
-    /**
-     * Реализуйте метод, который будет возвращать углы треугольника (в градусах) по возрастанию.
-     *
-     * <p>Входные параметры - длина сторон треугольника. Возвращаемое значение - массив с углами треугольника.
-     *
-     * <p>Если входные данные некорректны - метод должен возвращать пустой массив нулевой длины.
-     */
     static double[] getAngles(double a, double b, double c) {
-        //        Место для вашего кода
 
-        return null; // Заглушка. При реализации - удалить
+        if (!isDegenerate(a, b, c)) {
+            return new double[0];
+        }
+
+        double[] angelsMassive = new double[3];
+        angelsMassive[0] = Math.acos((b * b + c * c - a * a) / (2 * b * c)) * 180 / Math.PI;
+        angelsMassive[1] = Math.acos((a * a + c * c - b * b) / (2 * a * c)) * 180 / Math.PI;
+        angelsMassive[2] = Math.acos((b * b + a * a - c * c) / (2 * b * a)) * 180 / Math.PI;
+        Arrays.sort(angelsMassive);
+        return angelsMassive;
     }
 
-    /**
-     * Реализуйте метод, который будет возвращать длину радиуса вписанной в треугольник окружности.
-     *
-     * <p>Входные параметры - длина сторон треугольника.
-     *
-     * <p>Если входные данные некорректны - метод должен возвращать -1.
-     */
     static double getInscribedCircleRadius(double a, double b, double c) {
-        //        Место для вашего кода
 
-        return 0; // Заглушка. При реализации - удалить
+        if (!isDegenerate(a, b, c)) {
+            return -1;
+        }
+
+        return Math.sqrt((-a + b + c) * (a - b + c) * (a + b - c) / (4 * (a + b + c)));
     }
 
-    /**
-     * Реализуйте метод, который будет возвращать длину радиуса описанной вокруг треугольника окружности.
-     *
-     * <p>Входные параметры - длина сторон треугольника.
-     *
-     * <p>Если входные данные некорректны - метод должен возвращать -1.
-     */
     static double getCircumradius(double a, double b, double c) {
-        //        Место для вашего кода
 
-        return 0; // Заглушка. При реализации - удалить
+        if (!isDegenerate(a, b, c)) {
+            return -1;
+        }
+
+        double semiPerimeter = (a + b + c) / 2;
+
+        return (a * b * c) / (4 * Math.sqrt(semiPerimeter * (semiPerimeter - a) * (semiPerimeter - b) * (semiPerimeter - c)));
```

Comment:
Error: eckstyle] [ERROR] /home/runner/work/intensive-tasks-2024/intensive-tasks-2024/src/main/java/com/walking/intensive/chapter1/task5/Task5.java:120: Line is longer than 120 characters (found 126). [LineLength]